src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue26083.cs

```diff
@@ -0,0 +1,26 @@
+﻿#if WINDOWS || MACCATALYST //Window resizing behavior is only present on Mac Catalyst and Windows platforms.
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue26083 : _IssuesUITest
+	{
+		public Issue26083(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "UI not updating GridItemsLayout when Span becomes 1";
+
+		[Test]
+		[Category(UITestCategories.CollectionView)]
+		public void ItemsWrapGridShouldUpdateBasedOnCollectionViewSize()
+		{
+			App.WaitForElement("Button");
+			App.Tap("Button");
```

Comment:
Not mandatory, but could be interesting to verify a snapshot before tap the button and change the Span property.